### DISCUSSION TITLE
Many languages

### DISCUSSION THREAD

**Discussion by auronandace** [GyulyVGC/sniffnet#167] [Ideas]
I just read the interview you had here: https://console.substack.com/p/console-154#%C2%A7sniffnet and noticed the section about handling languages. You also mentioned about iced not currently supporting font fallback. Just thought I'd share some links for consideration.

https://github.com/pop-os/cosmic-text
https://github.com/AccessKit/accesskit

I don't know if you are already aware of them or if it is possible to use them to your advantage but I hope they might help.

Great interview, by the way, and thanks for your hard work on sniffnet!

**Comment by GyulyVGC** [comment]
Thank you very much for your interest and your advices!

I'm aware of `cosmic-text`. Iced is using it on a secondary branch at the moment, since they're still performing some improvements before officially releasing a new version integrating it.
That being said, I always prefer depending on the master branch of iced to have more stability.
They are moving fast and I think support for font fallback will finally come soon.

PS: I still remember you as one of the earliest adopters of Sniffnet, thanks for being here for so long 💯
